.github/workflows/generate-cve.yml

```diff
@@ -65,23 +65,31 @@ jobs:
         uses: actions/setup-go@v4.1.0
         with:
           cache: false
-          go-version: '^1.21.4'
+          go-version: '^1.23.3'
 
       - name: Generate NVD Feeds
         run: |
           cd fleet
           go mod download
           go run cmd/cve/generate.go --db_dir ./cvefeed --debug
 
-      - name: Generate goval-dictionary sqlite files
+      - name: Build goval-dictionary and ensure destination dir exists
         run: |
           cd goval-dictionary
           make build
-          ./goval-dictionary fetch amazon 1 --dbpath ../fleet/cvefeed/amzn_01.sqlite3
-          ./goval-dictionary fetch amazon 2 --dbpath ../fleet/cvefeed/amzn_02.sqlite3
-          ./goval-dictionary fetch amazon 2022 --dbpath ../fleet/cvefeed/amzn_2022.sqlite3
-          ./goval-dictionary fetch amazon 2023 --dbpath ../fleet/cvefeed/amzn_2023.sqlite3
-          xz ../fleet/cvefeed/*.sqlite3
+          mkdir -p ../fleet/cvefeed
+
+      - name: Pull Amazon Linux 1 vulnerabilities
+        run: for i in {1..3}; do goval-dictionary/goval-dictionary fetch amazon 1 --dbpath ./fleet/cvefeed/amzn_01.sqlite3 && s=0 && break || s=$? && sleep 2; done; (exit $s)
```

Comment:
2s seems short, but we can adjust as needed

Reply:
It's CloudFront blips so I'm reasonably confident that's enough. We'll find out soon enough :)